Source/WebKit/Platform/IPC/Decoder.h

```diff
@@ -149,6 +149,12 @@ class Decoder {
         return *this;
     }
 
+    Decoder& operator>>(std::optional<Attachment>& t)
```

Comment:
Do we really need this, since we have the Decoder::decode overload?

Reply:
Maybe not, I'll check

Reply:
Looks like it wasn't needed, I dropped it.